src/styles/menu-button.js

```diff
@@ -18,6 +18,10 @@ export default css`
     align-items: center;
   }
 
+  button.menu-item {
+    border: none;
```

Comment:
You can just move this into the class selector block above. Both elements should have identical styles when rendered. With the addition of `button` as a possibility, these properties also need to be added:

```css
  cursor: pointer;
  -webkit-appearance: none;
  appearance: none;
  border-radius: 0;
```

Reply:
thanks, just updated and also moved cursor up